### ISSUE TITLE
Option to not treat Markdown files as attachments

### ISSUE THREAD

**Issue by igorakkerman** [dy-sh/obsidian-consistent-attachments-and-links#73]
I use Markdown files as embeds, such as:
```md
Tool list:
![My tool 1](My%20Tool%201.md)
![My tool 2](My%20Tool%202.md)
```
to split up files and to include other notes, so that I don't have to navigate to a note to see its content. Unfortunately, this plugin treats Markdown files as attachments while they are separate notes.

Please add an option to exclude embedded Markdown files from being treated as attachments and being moved to the attachment folder. Thank you!

**Comment by mnaoumov** [comment]
I don't think we ever need to treat Markdown as an attachment. It was a bug fixed in #67

Properly released in `v1.0.11`

**Comment by igorakkerman** [comment]
@mnaoumov Thanks a lot!

**Comment by igorakkerman** [comment]
@mnaoumov I'm running v1.0.11`now but embedded Markdown files are still being moved to the attachments folder 🙁

**Comment by mnaoumov** [comment]
@igorakkerman thanks for your report. Found the cause and fixed in `1.1.0`

**Comment by igorakkerman** [comment]
Thank you for taking care, seems to be fixed now!
